agent/src/main/java/io/pyroscope/javaagent/JFRJDKProfilerDelegate.java

```diff
@@ -66,6 +95,57 @@ public synchronized void start() {
         }
     }
 
+    // this based on async-profiler parsing logic so we can things compatible
+    private static Duration parseDuration(String str, Duration defaultValue) {
```

Comment:
I am not 100% sure, but can you check if we can use `IntervalParser.parse` instead of new parser utility. If not, then let's move this function into `IntervalParser` or `Config` with a comment why we cant use the `IntervalParser.parse`